src/OrbitGl/MockBatcher.cpp

```diff
@@ -91,7 +91,8 @@ int MockBatcher::GetNumBoxes() const {
 
 // To check that everything is inside a rectangle, we just need to check the minimum and maximum
 // used coordinates.
-bool MockBatcher::IsEverythingInsideRectangle(Vec2 start, Vec2 end) const {
+bool MockBatcher::IsEverythingInsideRectangle(Vec2 start, Vec2 size) const {
```

Comment:
What about making this function taking a `Box` instead of 2 `Vec2`?

Reply:
I'm asking because this change is kind of "dangerous". The compiler will happily compile the code if someone provides `start` and `end` instead of `start` and `size`.

Reply:
Good point. We don't have a Rectangle definition anymore. I will think about this.

Reply:
The Box includes 4 vertices and it is not necessarily a Rectangle. And anyway creating one with MakeBox is still asking for start and size. I don't have a better idea so I will leave it as it is.